packages/kit/src/vite/index.js

```diff
@@ -2,7 +2,7 @@ import fs from 'fs';
 import path from 'path';
 import colors from 'kleur';
 import { svelte } from '@sveltejs/vite-plugin-svelte';
-import { searchForWorkspaceRoot } from 'vite';
+import { searchForWorkspaceRoot, version as vite_version } from 'vite';
```

Comment:
I don't think this works in Vite 2 though, `version` is only exported in Vite 3. Maybe we should check `vite/package.json` instead.

Reply:
oh, whoops. presumably that means mucking about with `import-meta-resolve` which is a bit of a pain. what if we `import * as vite from 'vite'` and then check for the existence of the `version` export? if it's there, check it, if not, assume it's version 2